packages/amesos2/src/Amesos2_MUMPS.cpp

```diff
@@ -50,6 +50,7 @@
 
 namespace Amesos2 {
 
+// TODO: How does TPETRA_INST_INT_INT influence an Epetra instantiation???
```

Comment:
It's a known issue that if you enable Epetra in Amesos2, then you must enable `GlobalOrdinal=int` in Tpetra.  See #2548.  This is unfortunate and Trilinos needs to fix it, but I think that's out of scope for this PR.